src/cpu/aarch64/injectors/jit_uni_eltwise_injector.hpp

```diff
@@ -79,11 +80,12 @@ bool is_supported(cpu_isa_t isa, alg_kind_t alg);
 
 } // namespace eltwise_injector
 
-template <cpu_isa_t isa>
 struct jit_uni_eltwise_injector_f32 {
-    using TReg = typename cpu_isa_traits<isa>::TReg;
-    using TRegS = typename cpu_isa_traits<isa>::TRegS;
-
+    typedef Xbyak_aarch64::ZReg TReg;
```

Comment:
I don't think this is the right approach. If we are specializing our class to SVE then we should explicitly use `ZReg` everywhere rather than aliasing `TReg`